src/ngrest/plugins/Injector.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace luya\admin\ngrest\plugins;
+
+use luya\admin\ngrest\base\Plugin;
+use luya\helpers\Html as HtmlHelper;
+
+/**
+ * Create a text input select for a given field.
+ *
+ * @author Bennet Klarhoelter <boehsermoe@me.com>
+ * @since 1.2.3
+ */
+class Injector extends Plugin
+{
+    /**
+     * @var string
+     */
+    public $dir;
+    
+    /**
+     * @var boolean Whether the value should be encoded after find by {{luya\helpers\Html::encode()}} or not.
+     */
+    public $encoding = true;
+    
+    /**
+     * @var array An array with options can be passed to the createListTag.
+     */
+    public $listOptions = [];
```

Comment:
@boehsermoe it would be nice to see a phpdoc example of "what" options can be passed and what it looks like when rendering.